src/docs/guides/contributing/reviewing-pull-requests.md

```diff
@@ -0,0 +1,23 @@
+# Reviewing pull requests
+
+Giving feedback on pull requests, i.e. reviewing them, is a very important contribution too. While it's mostly core contributors doing reviews, don't shy away from a review if you aren't one! Anybody can review pull requests, just core contributors can merge them.
+
+!!! tip
+    Reviews also show up as green blobs under your GitHub profile, did you know that? :)
+
+## Guidelines for reviewing code
+
+For general guidelines on reviewing code, see the [Dojo Library](https://orcharddojo.net/orchard-resources/CoreLibrary/DevelopmentGuidelines/CodeReview). Stack Overflow also has [some good principles](https://stackoverflow.blog/2019/09/30/how-to-make-good-code-reviews-better/).
+
+## Accessing the code of a pull request locally
+
+If the PR is coming from a fork, working with its code locally won't be as trivial as doing a `git checkout` on the branch of a core contributor. See [GitHub's docs on this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).
+
+## Tips on managing pull requests
+
+- [Here's a list](https://github.com/OrchardCMS/OrchardCore/pulls?q=is%3Apr+is%3Aopen+reviewed-by%3A%40me) of open PRs reviewed by you. Maybe check them because perhaps it's time to ping the authors since maybe they forgot to follow up? Or maybe you forgot to merge a PR? 
+- Add the "don't merge" label on PRs that are ready for review, approved by you and/or others, but you asked more people for feedback before it can be merged.
```

Comment:
In reality, contributors are currently unable to add any labels other than "Don't merge" label

Reply:
Hmm, strange. Could you please send a screenshot of this as you see it?

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/890230a8-778b-4d41-a94c-5c967d32895f)

Reply:
this is my screen

![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/b38f1490-4ea7-4105-b1c4-918d5521e675)

Reply:
I see, so you can't actually add any label, strange. I added a note on this.

Reply:
Yes, and as @hishamco mentioned earlier, it should be possible to specify a member of the review, but I can't do that here

Reply:
That's unfortunately just how GitHub works.